client/driver/docker_test.go

```diff
@@ -879,6 +880,10 @@ func TestDockerDriver_DNS(t *testing.T) {
 	if !reflect.DeepEqual(task.Config["dns_search_domains"], container.HostConfig.DNSSearch) {
 		t.Errorf("DNS Servers don't match.\nExpected:\n%s\nGot:\n%s\n", task.Config["dns_search_domains"], container.HostConfig.DNSSearch)
 	}
+
+	if !reflect.DeepEqual(task.Config["dns_options"], container.HostConfig.DNSOptions) {
+		t.Errorf("DNS Servers don't match.\nExpected:\n%s\nGot:\n%s\n", task.Config["dns_options"], container.HostConfig.DNSOptions)
```

Comment:
Hardly worth mentioning but looks like there's a copy/paste bug in there. Probably want to change "Servers" to "Options".